### DISCUSSION TITLE
How to respond to an HTTP request

### DISCUSSION THREAD

**Discussion by MasterMong** [totaljs/flow#116] [General]
![Screenshot from 2024-04-04 12-05-24](https://github.com/totaljs/flow/assets/19867358/8e872980-d469-461f-92ca-a9c77081bacb)

**Comment by petersirka** [comment]
The `Response` component contains `Path` in the settings, so the response value (output) is read from this property of incoming message data.

![20240404-083314](https://github.com/totaljs/flow/assets/2414252/453e8fa1-45c4-45f2-b0bd-e64be968fd6b)

In your case, just remove the `path` value, and everything will work as you expect.

**Reply by MasterMong** [reply]
Wow it work now. thank you so much for make such excellent tool.
